packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -13,6 +13,11 @@ Source101: config-bottlerocket-aws
 Source102: config-bottlerocket-metal
 Source103: config-bottlerocket-vmware
 
+# This list of FIPS modules is extracted from /etc/fipsmodules in the initramfs
+# after placing AL2023 in FIPS mode.
+Source200: fipsmodules-x86_64
+Source201: fipsmodules-aarch64
```

Comment:
This is my least favorite part of this change, but I'm not sure how else to generate this list of files.

The `dracut` package in AL23 has a list of modules in `/usr/lib/dracut/modules.d/01fips/module-setup.sh`, but some are architecture-specific and it's not really correct to include all of those. That list gets passed to dracut's `instmods` function which interacts with the available modules on the running system.

For what it's worth I don't expect this list to change much across the life of the 6.1 kernel - worst case only when it needed to be recertified.